### PR TITLE
app-admin/ulogd: verbump to 2.0.5_p20160331

### PR DESCRIPTION
**Pull request by Coacher**

### CODE REVIEW

**Comment by Coacher** [comment]
CC @gentoo/proxy-maint

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	doman ${PN}.8
+
+	use doc && dodoc doc/${PN}.{dvi,ps,txt}
+	use mysql && dodoc doc/mysql-*.sql
+	use postgres && dodoc doc/pgsql-*.sql
+	use sqlite && dodoc doc/sqlite3.table
+
+	insinto /etc
+	doins ${PN}.conf
+	fowners root:ulogd /etc/${PN}.conf
```
Comment (monsieurp) [inline app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild]: `ulogd` = ${PN} in this context AFAICT.

File: app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+	fperms 640 /etc/${PN}.conf
+
+	newinitd "${FILESDIR}/${PN}.init" ${PN}
+	systemd_dounit "${FILESDIR}/${PN}.service"
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}/${PN}.logrotate" ${PN}
+
+	diropts -o ulogd -g ulogd
```
Comment (monsieurp) [inline app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild]: Ditto.

File: app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+	keepdir /var/log/ulogd
```
Comment (monsieurp) [inline app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild]: Same.

File: app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild (hunk 1 above)
Comment (Coacher) [inline app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild]: I don't want to use ${PN} in the context of groups/permissions.
I'd like it to be visually distinct from paths, etc.

File: app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild (hunk 2 above)
Comment (Coacher) [inline app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild]: Ditto.

File: app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild (hunk 3 above)
Comment (Coacher) [inline app-admin/ulogd/ulogd-2.0.5_p20160331.ebuild]: Here I agree it can be changed to ${PN}

**Comment by Coacher** [comment]
Thank you very much, @monsieurp!

**Pull request closed without merging**